### ISSUE TITLE
Looks interesting, but I'm apparently not able to get the chrome extension working

### ISSUE THREAD

**Issue by Skrattoune** [ofzza/github-projects-extension#4]
I wasn't able to find anywhere a kind of guideline to tell me how I should be using the Chrome extension (PC, Windows 10)

Where should I go ?

**Comment by ofzza** [comment]
There should have been a `?` info token next to all new/editing labels that explain this, but as GitHub updated that part of their page this got broken!

It is fixed now in version 2.0.8 which now awaiting to be published on Chrome store:
https://chrome.google.com/webstore/detail/github-projects-plus/pkkhkgaamkjaepakanehpgbifoljadnl
... or can be downloaded unpacked here:
https://github.com/ofzza/github-projects-extension/raw/master/versions/prod/dist.githubcom.v2.0.8.zip

After updating, you should see a `?` info token which on hover show instructions on how to create and use `@special` and `#type` labels:
![image](https://user-images.githubusercontent.com/4250750/69809116-5e938600-11e9-11ea-9a63-cef57e9a832b.png)
![image](https://user-images.githubusercontent.com/4250750/69809135-66532a80-11e9-11ea-8061-5bda4e1f428b.png)

Basically:
- all labels with names starting with `@` will be displayed on the right border of issue cards as little circles
- all labels with names starting with a `#` will be displayed on the right border of issue cards as a thin line.

These are meant to be used for labels that signify status (such as "@blocked", or "@ready") or assignment (such as "#team-API", or "#team-UI") which need not clutter the main labels list in the already information dense issue cards.

**Comment by Skrattoune** [comment]
Thanks a lot for the support.
I'll wait a bit for chrome to move to the next version.

Best regards

**Comment by ofzza** [comment]
Version 2.0.8 is up on the Chrome store.
